darshan_logs/hdf5_diagonal_write_only/README.md

```diff
@@ -0,0 +1,22 @@
+Set of example darshan logs that contain `H5D` and/or `H5F` module data
+to allow for "ground truth" verification for various data aggregation
+functions in the new Python summary report.
+
```

Comment:
Do you mind adding a quick note here that these logs were generated by Python test programs using h5py?

I know you can see that by following the github link to the test programs, but it might be helpful to see at a glance since that could be relevant from a runtime perspective.

Reply:
That's a good point, I'll add a comment about that so it's more clear.

Reply:
Okay, I've added a comment for this.